scripts/cc-cli/src/commands/unbond.ts

```diff
@@ -23,8 +18,8 @@ async function unbondAction(options: OptionValues) {
   checkAmount(options);
 
   // Build account
-  const seed = getSeedFromOptions(options);
-  const stash = initKeyringPair(seed);
+  const stashSeed = await getStashSeedFromEnvOrPrompt();
+  const stash = initKeyringPair(stashSeed);
```

Comment:
From https://crates.parity.io/pallet_staking/enum.Call.html#variant.unbond

`The dispatch origin for this call must be Signed by the controller, not the stash.`

Reply:
done